examples/amp-onetap-google/iframe.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <script src="https://accounts.google.com/gsi/client"></script>
+  <style>
+    #credential_picker_container {
+      position: fixed !important;
+      right: 0 !important;
+      top: 60px !important;
+    }
+  </style>
+  <script>
+    let responseReturned = false;
+
+    function notifyParent(message) {
+      window.parent && window.parent.postMessage(message, '*');
```

Comment:
Put a sentinel in the message to differentiate message types - see amp-iframe for example https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/0.1/iframe-api/messenger.js

Reply:
Added